src/openrct2/ride/RideStringIds.h

```diff
@@ -0,0 +1,208 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2024 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include "../common.h"
+
+enum : StringId
+{
+    // Coasters
+    STR_RIDE_DESCRIPTION_AIR_POWERED_VERTICAL_COASTER = 587,
```

Comment:
It seems a bit odd to sort the description before the name, can you reverse that?